Installation from npm fails

```
$ npm install -g go-ipfs
npm ERR! path /Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/bin/ipfs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/bin/ipfs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

$ npm -v
5.5.1

$ node -v
v8.8.1
```

Thanks @dryajov @VictorBjelkholm.  

@dryajov after you suggested this, I tried over a different connection, as well as over my client's network.  I've tried a few times.  Everything seems to be coming from npmjs.org.  Here's my `npm config list`:

```
metrics-registry = "https://registry.npmjs.org/"
scope = ""
user-agent = "npm/5.5.1 node/v8.8.1 darwin x64"
```

I do have nvm installed using Homebrew, which I understand is discouraged, but this hadn't caused me any problems since I started with Node.

@VictorBjelkholm here's the output.

```sh
$ npm install -g go-ipfs --verbose

npm info it worked if it ends with ok
npm verb cli [ '/Users/danny/.nvm/versions/node/v8.8.1/bin/node',
npm verb cli   '/Users/danny/.nvm/versions/node/v8.8.1/bin/npm',
npm verb cli   'install',
npm verb cli   '-g',
npm verb cli   'go-ipfs',
npm verb cli   '--verbose' ]
npm info using npm@5.5.1
npm info using node@v8.8.1
npm verb npm-session bc9d2d1ea9ebcc54
npm http fetch GET 200 https://registry.npmjs.org/go-ipfs 13ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/go-platform 36ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/go-ipfs-dep 38ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/gunzip-maybe 37ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tar-fs 37ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/unzip 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/request 333ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/is-gzip 10ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/peek-stream 10ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/browserify-zlib 12ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/is-deflate 12ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/pumpify 10ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/through2 11ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/pako 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/duplexify 1ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/end-of-stream 9ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/readable-stream 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/stream-shift 9ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/inherits 62ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/once 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/wrappy 3ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/isarray 11ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/safe-buffer 12ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/string_decoder 12ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/util-deprecate 13ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/core-util-is 58ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/process-nextick-args 239ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/xtend 3ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/pump 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/aws4 18ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/aws-sign2 18ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/caseless 20ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/combined-stream 20ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/forever-agent 23ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/extend 23ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/form-data 22ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/hawk 23ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/har-validator 24ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/http-signature 26ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/is-typedarray 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/isstream 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/json-stringify-safe 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/mime-types 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/performance-now 15ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/oauth-sign 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/qs 16ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/stringstream 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tough-cookie 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tunnel-agent 17ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/uuid 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/delayed-stream 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/asynckit 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/mime-db 1ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/har-schema 3ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/ajv 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/co 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/json-schema-traverse 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/fast-deep-equal 7ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/fast-json-stable-stringify 7ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/hoek 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/boom 7ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/cryptiles 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/sntp 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/assert-plus 4ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/jsprim 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/sshpk 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/json-schema 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/extsprintf 5ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/verror 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/asn1 13ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/dashdash 14ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/jsbn 13ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/getpass 15ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tweetnacl 13ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/ecc-jsbn 14ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/bcrypt-pbkdf 15ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/punycode 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/chownr 4ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tar-stream 5ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/mkdirp 59ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/minimist 65ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/bl 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/pullstream 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/binary 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/setimmediate 8ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/match-stream 7ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fstream 60ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/chainsaw 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/buffers 3ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/traverse 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/graceful-fs 3ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/rimraf 50ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/natives 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/glob 2ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/inflight 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/minimatch 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/path-is-absolute 6ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fs.realpath 56ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/brace-expansion 58ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/concat-map 4ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/balanced-match 70ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/over 3ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/slice-stream 4ms (from cache)
npm verb correctMkdir /Users/danny/.npm/_locks correctMkdir not in flight; initializing
npm verb lock using /Users/danny/.npm/_locks/staging-c25489efd2f142c4.lock for /Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/.staging
npm info lifecycle asn1@0.2.3~preinstall: asn1@0.2.3
npm info lifecycle assert-plus@1.0.0~preinstall: assert-plus@1.0.0
npm info lifecycle asynckit@0.4.0~preinstall: asynckit@0.4.0
npm info lifecycle aws-sign2@0.7.0~preinstall: aws-sign2@0.7.0
npm info lifecycle aws4@1.6.0~preinstall: aws4@1.6.0
npm info lifecycle balanced-match@1.0.0~preinstall: balanced-match@1.0.0
npm info lifecycle buffers@0.1.1~preinstall: buffers@0.1.1
npm info lifecycle caseless@0.12.0~preinstall: caseless@0.12.0
npm info lifecycle chownr@1.0.1~preinstall: chownr@1.0.1
npm info lifecycle co@4.6.0~preinstall: co@4.6.0
npm info lifecycle concat-map@0.0.1~preinstall: concat-map@0.0.1
npm info lifecycle brace-expansion@1.1.8~preinstall: brace-expansion@1.1.8
npm info lifecycle core-util-is@1.0.2~preinstall: core-util-is@1.0.2
npm info lifecycle dashdash@1.14.1~preinstall: dashdash@1.14.1
npm info lifecycle delayed-stream@1.0.0~preinstall: delayed-stream@1.0.0
npm info lifecycle combined-stream@1.0.5~preinstall: combined-stream@1.0.5
npm info lifecycle extend@3.0.1~preinstall: extend@3.0.1
npm info lifecycle extsprintf@1.3.0~preinstall: extsprintf@1.3.0
npm info lifecycle fast-deep-equal@1.0.0~preinstall: fast-deep-equal@1.0.0
npm info lifecycle fast-json-stable-stringify@2.0.0~preinstall: fast-json-stable-stringify@2.0.0
npm info lifecycle forever-agent@0.6.1~preinstall: forever-agent@0.6.1
npm info lifecycle fs.realpath@1.0.0~preinstall: fs.realpath@1.0.0
npm info lifecycle getpass@0.1.7~preinstall: getpass@0.1.7
npm info lifecycle go-platform@1.0.0~preinstall: go-platform@1.0.0
npm info lifecycle har-schema@2.0.0~preinstall: har-schema@2.0.0
npm info lifecycle hoek@4.2.0~preinstall: hoek@4.2.0
npm info lifecycle boom@4.3.1~preinstall: boom@4.3.1
npm info lifecycle boom@5.2.0~preinstall: boom@5.2.0
npm info lifecycle cryptiles@3.1.2~preinstall: cryptiles@3.1.2
npm info lifecycle inherits@2.0.3~preinstall: inherits@2.0.3
npm info lifecycle is-deflate@1.0.0~preinstall: is-deflate@1.0.0
npm info lifecycle is-gzip@1.0.0~preinstall: is-gzip@1.0.0
npm info lifecycle is-typedarray@1.0.0~preinstall: is-typedarray@1.0.0
npm info lifecycle isarray@1.0.0~preinstall: isarray@1.0.0
npm info lifecycle isstream@0.1.2~preinstall: isstream@0.1.2
npm info lifecycle jsbn@0.1.1~preinstall: jsbn@0.1.1
npm info lifecycle ecc-jsbn@0.1.1~preinstall: ecc-jsbn@0.1.1
npm info lifecycle json-schema@0.2.3~preinstall: json-schema@0.2.3
npm info lifecycle json-schema-traverse@0.3.1~preinstall: json-schema-traverse@0.3.1
npm info lifecycle ajv@5.3.0~preinstall: ajv@5.3.0
npm info lifecycle har-validator@5.0.3~preinstall: har-validator@5.0.3
npm info lifecycle json-stringify-safe@5.0.1~preinstall: json-stringify-safe@5.0.1
npm info lifecycle isarray@0.0.1~preinstall: isarray@0.0.1
npm info lifecycle string_decoder@0.10.31~preinstall: string_decoder@0.10.31
npm info lifecycle readable-stream@1.0.34~preinstall: readable-stream@1.0.34
npm info lifecycle match-stream@0.0.2~preinstall: match-stream@0.0.2
npm info lifecycle mime-db@1.30.0~preinstall: mime-db@1.30.0
npm info lifecycle mime-types@2.1.17~preinstall: mime-types@2.1.17
npm info lifecycle form-data@2.3.1~preinstall: form-data@2.3.1
npm info lifecycle minimatch@3.0.4~preinstall: minimatch@3.0.4
npm info lifecycle go-ipfs@0.4.11~preinstall: go-ipfs@0.4.11
npm info lifecycle go-ipfs-dep@0.4.11~preinstall: go-ipfs-dep@0.4.11
npm info lifecycle gunzip-maybe@1.4.1~preinstall: gunzip-maybe@1.4.1
npm info lifecycle peek-stream@1.1.2~preinstall: peek-stream@1.1.2
npm info lifecycle through2@2.0.3~preinstall: through2@2.0.3
npm info lifecycle tar-fs@1.16.0~preinstall: tar-fs@1.16.0
npm info lifecycle tar-stream@1.5.4~preinstall: tar-stream@1.5.4
npm info lifecycle xtend@4.0.1~preinstall: xtend@4.0.1
npm info lifecycle pumpify@1.3.5~preinstall: pumpify@1.3.5
npm info lifecycle pump@1.0.2~preinstall: pump@1.0.2
npm info lifecycle unzip@0.1.11~preinstall: unzip@0.1.11
npm info lifecycle fstream@0.1.31~preinstall: fstream@0.1.31
npm info lifecycle rimraf@2.6.2~preinstall: rimraf@2.6.2
npm info lifecycle glob@7.1.2~preinstall: glob@7.1.2
npm info lifecycle inflight@1.0.6~preinstall: inflight@1.0.6
npm info lifecycle duplexify@3.5.1~preinstall: duplexify@3.5.1
npm info lifecycle end-of-stream@1.4.0~preinstall: end-of-stream@1.4.0
npm info lifecycle once@1.4.0~preinstall: once@1.4.0
npm info lifecycle wrappy@1.0.2~preinstall: wrappy@1.0.2
npm info lifecycle request@2.83.0~preinstall: request@2.83.0
npm info lifecycle http-signature@1.2.0~preinstall: http-signature@1.2.0
npm info lifecycle jsprim@1.4.1~preinstall: jsprim@1.4.1
npm info lifecycle verror@1.10.0~preinstall: verror@1.10.0
npm info lifecycle uuid@3.1.0~preinstall: uuid@3.1.0
npm info lifecycle bl@1.2.1~preinstall: bl@1.2.1
npm info lifecycle readable-stream@2.3.3~preinstall: readable-stream@2.3.3
npm info lifecycle util-deprecate@1.0.2~preinstall: util-deprecate@1.0.2
npm info lifecycle readable-stream@1.0.34~preinstall: readable-stream@1.0.34
npm info lifecycle string_decoder@0.10.31~preinstall: string_decoder@0.10.31
npm info lifecycle isarray@0.0.1~preinstall: isarray@0.0.1
npm info lifecycle sshpk@1.13.1~preinstall: sshpk@1.13.1
npm info lifecycle bcrypt-pbkdf@1.0.1~preinstall: bcrypt-pbkdf@1.0.1
npm info lifecycle tweetnacl@0.14.5~preinstall: tweetnacl@0.14.5
npm info lifecycle tunnel-agent@0.6.0~preinstall: tunnel-agent@0.6.0
npm info lifecycle binary@0.3.0~preinstall: binary@0.3.0
npm info lifecycle chainsaw@0.1.0~preinstall: chainsaw@0.1.0
npm info lifecycle traverse@0.3.9~preinstall: traverse@0.3.9
npm info lifecycle tough-cookie@2.3.3~preinstall: tough-cookie@2.3.3
npm info lifecycle stringstream@0.0.5~preinstall: stringstream@0.0.5
npm info lifecycle string_decoder@1.0.3~preinstall: string_decoder@1.0.3
npm info lifecycle stream-shift@1.0.0~preinstall: stream-shift@1.0.0
npm info lifecycle hawk@6.0.2~preinstall: hawk@6.0.2
npm info lifecycle sntp@2.1.0~preinstall: sntp@2.1.0
npm info lifecycle pullstream@0.4.1~preinstall: pullstream@0.4.1
npm info lifecycle slice-stream@1.0.0~preinstall: slice-stream@1.0.0
npm info lifecycle readable-stream@1.0.34~preinstall: readable-stream@1.0.34
npm info lifecycle string_decoder@0.10.31~preinstall: string_decoder@0.10.31
npm info lifecycle isarray@0.0.1~preinstall: isarray@0.0.1
npm info lifecycle setimmediate@1.0.5~preinstall: setimmediate@1.0.5
npm info lifecycle safe-buffer@5.1.1~preinstall: safe-buffer@5.1.1
npm info lifecycle qs@6.5.1~preinstall: qs@6.5.1
npm info lifecycle punycode@1.4.1~preinstall: punycode@1.4.1
npm info lifecycle readable-stream@1.0.34~preinstall: readable-stream@1.0.34
npm info lifecycle string_decoder@0.10.31~preinstall: string_decoder@0.10.31
npm info lifecycle isarray@0.0.1~preinstall: isarray@0.0.1
npm info lifecycle process-nextick-args@1.0.7~preinstall: process-nextick-args@1.0.7
npm info lifecycle performance-now@2.1.0~preinstall: performance-now@2.1.0
npm info lifecycle path-is-absolute@1.0.1~preinstall: path-is-absolute@1.0.1
npm info lifecycle browserify-zlib@0.1.4~preinstall: browserify-zlib@0.1.4
npm info lifecycle pako@0.2.9~preinstall: pako@0.2.9
npm info lifecycle over@0.0.5~preinstall: over@0.0.5
npm info lifecycle oauth-sign@0.8.2~preinstall: oauth-sign@0.8.2
npm info lifecycle graceful-fs@3.0.11~preinstall: graceful-fs@3.0.11
npm info lifecycle natives@1.1.0~preinstall: natives@1.1.0
npm info lifecycle mkdirp@0.5.1~preinstall: mkdirp@0.5.1
npm info lifecycle minimist@0.0.8~preinstall: minimist@0.0.8
npm info linkStuff asn1@0.2.3
npm verb linkBins asn1@0.2.3
npm verb linkMans asn1@0.2.3
npm info linkStuff assert-plus@1.0.0
npm verb linkBins assert-plus@1.0.0
npm verb linkMans assert-plus@1.0.0
npm info linkStuff asynckit@0.4.0
npm verb linkBins asynckit@0.4.0
npm verb linkMans asynckit@0.4.0
npm info linkStuff aws-sign2@0.7.0
npm verb linkBins aws-sign2@0.7.0
npm verb linkMans aws-sign2@0.7.0
npm info linkStuff aws4@1.6.0
npm verb linkBins aws4@1.6.0
npm verb linkMans aws4@1.6.0
npm info linkStuff balanced-match@1.0.0
npm verb linkBins balanced-match@1.0.0
npm verb linkMans balanced-match@1.0.0
npm info linkStuff buffers@0.1.1
npm verb linkBins buffers@0.1.1
npm verb linkMans buffers@0.1.1
npm info linkStuff caseless@0.12.0
npm verb linkBins caseless@0.12.0
npm verb linkMans caseless@0.12.0
npm info linkStuff chownr@1.0.1
npm verb linkBins chownr@1.0.1
npm verb linkMans chownr@1.0.1
npm info linkStuff co@4.6.0
npm verb linkBins co@4.6.0
npm verb linkMans co@4.6.0
npm info linkStuff concat-map@0.0.1
npm verb linkBins concat-map@0.0.1
npm verb linkMans concat-map@0.0.1
npm info linkStuff brace-expansion@1.1.8
npm verb linkBins brace-expansion@1.1.8
npm verb linkMans brace-expansion@1.1.8
npm info linkStuff core-util-is@1.0.2
npm verb linkBins core-util-is@1.0.2
npm verb linkMans core-util-is@1.0.2
npm info linkStuff dashdash@1.14.1
npm verb linkBins dashdash@1.14.1
npm verb linkMans dashdash@1.14.1
npm info linkStuff delayed-stream@1.0.0
npm verb linkBins delayed-stream@1.0.0
npm verb linkMans delayed-stream@1.0.0
npm info linkStuff combined-stream@1.0.5
npm verb linkBins combined-stream@1.0.5
npm verb linkMans combined-stream@1.0.5
npm info linkStuff extend@3.0.1
npm verb linkBins extend@3.0.1
npm verb linkMans extend@3.0.1
npm info linkStuff extsprintf@1.3.0
npm verb linkBins extsprintf@1.3.0
npm verb linkMans extsprintf@1.3.0
npm info linkStuff fast-deep-equal@1.0.0
npm verb linkBins fast-deep-equal@1.0.0
npm verb linkMans fast-deep-equal@1.0.0
npm info linkStuff fast-json-stable-stringify@2.0.0
npm verb linkBins fast-json-stable-stringify@2.0.0
npm verb linkMans fast-json-stable-stringify@2.0.0
npm info linkStuff forever-agent@0.6.1
npm verb linkBins forever-agent@0.6.1
npm verb linkMans forever-agent@0.6.1
npm info linkStuff fs.realpath@1.0.0
npm verb linkBins fs.realpath@1.0.0
npm verb linkMans fs.realpath@1.0.0
npm info linkStuff getpass@0.1.7
npm verb linkBins getpass@0.1.7
npm verb linkMans getpass@0.1.7
npm info linkStuff go-platform@1.0.0
npm verb linkBins go-platform@1.0.0
npm verb linkBins [ { 'go-platform': './cli.js' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans go-platform@1.0.0
npm info linkStuff har-schema@2.0.0
npm verb linkBins har-schema@2.0.0
npm verb linkMans har-schema@2.0.0
npm info linkStuff hoek@4.2.0
npm verb linkBins hoek@4.2.0
npm verb linkMans hoek@4.2.0
npm info linkStuff boom@4.3.1
npm verb linkBins boom@4.3.1
npm verb linkMans boom@4.3.1
npm info linkStuff boom@5.2.0
npm verb linkBins boom@5.2.0
npm verb linkMans boom@5.2.0
npm info linkStuff cryptiles@3.1.2
npm verb linkBins cryptiles@3.1.2
npm verb linkMans cryptiles@3.1.2
npm info linkStuff inherits@2.0.3
npm verb linkBins inherits@2.0.3
npm verb linkMans inherits@2.0.3
npm info linkStuff is-deflate@1.0.0
npm verb linkBins is-deflate@1.0.0
npm verb linkMans is-deflate@1.0.0
npm info linkStuff is-gzip@1.0.0
npm verb linkBins is-gzip@1.0.0
npm verb linkMans is-gzip@1.0.0
npm info linkStuff is-typedarray@1.0.0
npm verb linkBins is-typedarray@1.0.0
npm verb linkMans is-typedarray@1.0.0
npm info linkStuff isarray@1.0.0
npm verb linkBins isarray@1.0.0
npm verb linkMans isarray@1.0.0
npm info linkStuff isstream@0.1.2
npm verb linkBins isstream@0.1.2
npm verb linkMans isstream@0.1.2
npm info linkStuff jsbn@0.1.1
npm verb linkBins jsbn@0.1.1
npm verb linkMans jsbn@0.1.1
npm info linkStuff ecc-jsbn@0.1.1
npm verb linkBins ecc-jsbn@0.1.1
npm verb linkMans ecc-jsbn@0.1.1
npm info linkStuff json-schema@0.2.3
npm verb linkBins json-schema@0.2.3
npm verb linkMans json-schema@0.2.3
npm info linkStuff json-schema-traverse@0.3.1
npm verb linkBins json-schema-traverse@0.3.1
npm verb linkMans json-schema-traverse@0.3.1
npm info linkStuff ajv@5.3.0
npm verb linkBins ajv@5.3.0
npm verb linkMans ajv@5.3.0
npm info linkStuff har-validator@5.0.3
npm verb linkBins har-validator@5.0.3
npm verb linkMans har-validator@5.0.3
npm info linkStuff json-stringify-safe@5.0.1
npm verb linkBins json-stringify-safe@5.0.1
npm verb linkMans json-stringify-safe@5.0.1
npm info linkStuff isarray@0.0.1
npm verb linkBins isarray@0.0.1
npm verb linkMans isarray@0.0.1
npm info linkStuff string_decoder@0.10.31
npm verb linkBins string_decoder@0.10.31
npm verb linkMans string_decoder@0.10.31
npm info linkStuff readable-stream@1.0.34
npm verb linkBins readable-stream@1.0.34
npm verb linkMans readable-stream@1.0.34
npm info linkStuff match-stream@0.0.2
npm verb linkBins match-stream@0.0.2
npm verb linkMans match-stream@0.0.2
npm info linkStuff mime-db@1.30.0
npm verb linkBins mime-db@1.30.0
npm verb linkMans mime-db@1.30.0
npm info linkStuff mime-types@2.1.17
npm verb linkBins mime-types@2.1.17
npm verb linkMans mime-types@2.1.17
npm info linkStuff form-data@2.3.1
npm verb linkBins form-data@2.3.1
npm verb linkMans form-data@2.3.1
npm info linkStuff minimatch@3.0.4
npm verb linkBins minimatch@3.0.4
npm verb linkMans minimatch@3.0.4
npm info linkStuff minimist@0.0.8
npm verb linkBins minimist@0.0.8
npm verb linkMans minimist@0.0.8
npm info linkStuff mkdirp@0.5.1
npm verb linkBins mkdirp@0.5.1
npm verb linkBins [ { mkdirp: 'bin/cmd.js' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans mkdirp@0.5.1
npm info linkStuff natives@1.1.0
npm verb linkBins natives@1.1.0
npm verb linkMans natives@1.1.0
npm info linkStuff graceful-fs@3.0.11
npm verb linkBins graceful-fs@3.0.11
npm verb linkMans graceful-fs@3.0.11
npm info linkStuff oauth-sign@0.8.2
npm verb linkBins oauth-sign@0.8.2
npm verb linkMans oauth-sign@0.8.2
npm info linkStuff over@0.0.5
npm verb linkBins over@0.0.5
npm verb linkMans over@0.0.5
npm info linkStuff pako@0.2.9
npm verb linkBins pako@0.2.9
npm verb linkMans pako@0.2.9
npm info linkStuff browserify-zlib@0.1.4
npm verb linkBins browserify-zlib@0.1.4
npm verb linkMans browserify-zlib@0.1.4
npm info linkStuff path-is-absolute@1.0.1
npm verb linkBins path-is-absolute@1.0.1
npm verb linkMans path-is-absolute@1.0.1
npm info linkStuff performance-now@2.1.0
npm verb linkBins performance-now@2.1.0
npm verb linkMans performance-now@2.1.0
npm info linkStuff process-nextick-args@1.0.7
npm verb linkBins process-nextick-args@1.0.7
npm verb linkMans process-nextick-args@1.0.7
npm info linkStuff isarray@0.0.1
npm verb linkBins isarray@0.0.1
npm verb linkMans isarray@0.0.1
npm info linkStuff string_decoder@0.10.31
npm verb linkBins string_decoder@0.10.31
npm verb linkMans string_decoder@0.10.31
npm info linkStuff readable-stream@1.0.34
npm verb linkBins readable-stream@1.0.34
npm verb linkMans readable-stream@1.0.34
npm info linkStuff punycode@1.4.1
npm verb linkBins punycode@1.4.1
npm verb linkMans punycode@1.4.1
npm info linkStuff qs@6.5.1
npm verb linkBins qs@6.5.1
npm verb linkMans qs@6.5.1
npm info linkStuff safe-buffer@5.1.1
npm verb linkBins safe-buffer@5.1.1
npm verb linkMans safe-buffer@5.1.1
npm info linkStuff setimmediate@1.0.5
npm verb linkBins setimmediate@1.0.5
npm verb linkMans setimmediate@1.0.5
npm info linkStuff isarray@0.0.1
npm verb linkBins isarray@0.0.1
npm verb linkMans isarray@0.0.1
npm info linkStuff string_decoder@0.10.31
npm verb linkBins string_decoder@0.10.31
npm verb linkMans string_decoder@0.10.31
npm info linkStuff readable-stream@1.0.34
npm verb linkBins readable-stream@1.0.34
npm verb linkMans readable-stream@1.0.34
npm info linkStuff slice-stream@1.0.0
npm verb linkBins slice-stream@1.0.0
npm verb linkMans slice-stream@1.0.0
npm info linkStuff pullstream@0.4.1
npm verb linkBins pullstream@0.4.1
npm verb linkMans pullstream@0.4.1
npm info linkStuff sntp@2.1.0
npm verb linkBins sntp@2.1.0
npm verb linkMans sntp@2.1.0
npm info linkStuff hawk@6.0.2
npm verb linkBins hawk@6.0.2
npm verb linkMans hawk@6.0.2
npm info linkStuff stream-shift@1.0.0
npm verb linkBins stream-shift@1.0.0
npm verb linkMans stream-shift@1.0.0
npm info linkStuff string_decoder@1.0.3
npm verb linkBins string_decoder@1.0.3
npm verb linkMans string_decoder@1.0.3
npm info linkStuff stringstream@0.0.5
npm verb linkBins stringstream@0.0.5
npm verb linkMans stringstream@0.0.5
npm info linkStuff tough-cookie@2.3.3
npm verb linkBins tough-cookie@2.3.3
npm verb linkMans tough-cookie@2.3.3
npm info linkStuff traverse@0.3.9
npm verb linkBins traverse@0.3.9
npm verb linkMans traverse@0.3.9
npm info linkStuff chainsaw@0.1.0
npm verb linkBins chainsaw@0.1.0
npm verb linkMans chainsaw@0.1.0
npm info linkStuff binary@0.3.0
npm verb linkBins binary@0.3.0
npm verb linkMans binary@0.3.0
npm info linkStuff tunnel-agent@0.6.0
npm verb linkBins tunnel-agent@0.6.0
npm verb linkMans tunnel-agent@0.6.0
npm info linkStuff tweetnacl@0.14.5
npm verb linkBins tweetnacl@0.14.5
npm verb linkMans tweetnacl@0.14.5
npm info linkStuff bcrypt-pbkdf@1.0.1
npm verb linkBins bcrypt-pbkdf@1.0.1
npm verb linkMans bcrypt-pbkdf@1.0.1
npm info linkStuff sshpk@1.13.1
npm verb linkBins sshpk@1.13.1
npm verb linkBins [ { 'sshpk-conv': 'bin/sshpk-conv',
npm verb linkBins     'sshpk-sign': 'bin/sshpk-sign',
npm verb linkBins     'sshpk-verify': 'bin/sshpk-verify' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans sshpk@1.13.1
npm info linkStuff isarray@0.0.1
npm verb linkBins isarray@0.0.1
npm verb linkMans isarray@0.0.1
npm info linkStuff string_decoder@0.10.31
npm verb linkBins string_decoder@0.10.31
npm verb linkMans string_decoder@0.10.31
npm info linkStuff readable-stream@1.0.34
npm verb linkBins readable-stream@1.0.34
npm verb linkMans readable-stream@1.0.34
npm info linkStuff util-deprecate@1.0.2
npm verb linkBins util-deprecate@1.0.2
npm verb linkMans util-deprecate@1.0.2
npm info linkStuff readable-stream@2.3.3
npm verb linkBins readable-stream@2.3.3
npm verb linkMans readable-stream@2.3.3
npm info linkStuff bl@1.2.1
npm verb linkBins bl@1.2.1
npm verb linkMans bl@1.2.1
npm info linkStuff uuid@3.1.0
npm verb linkBins uuid@3.1.0
npm verb linkBins [ { uuid: './bin/uuid' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans uuid@3.1.0
npm info linkStuff verror@1.10.0
npm verb linkBins verror@1.10.0
npm verb linkMans verror@1.10.0
npm info linkStuff jsprim@1.4.1
npm verb linkBins jsprim@1.4.1
npm verb linkMans jsprim@1.4.1
npm info linkStuff http-signature@1.2.0
npm verb linkBins http-signature@1.2.0
npm verb linkMans http-signature@1.2.0
npm info linkStuff request@2.83.0
npm verb linkBins request@2.83.0
npm verb linkMans request@2.83.0
npm info linkStuff wrappy@1.0.2
npm verb linkBins wrappy@1.0.2
npm verb linkMans wrappy@1.0.2
npm info linkStuff once@1.4.0
npm verb linkBins once@1.4.0
npm verb linkMans once@1.4.0
npm info linkStuff end-of-stream@1.4.0
npm verb linkBins end-of-stream@1.4.0
npm verb linkMans end-of-stream@1.4.0
npm info linkStuff duplexify@3.5.1
npm verb linkBins duplexify@3.5.1
npm verb linkMans duplexify@3.5.1
npm info linkStuff inflight@1.0.6
npm verb linkBins inflight@1.0.6
npm verb linkMans inflight@1.0.6
npm info linkStuff glob@7.1.2
npm verb linkBins glob@7.1.2
npm verb linkMans glob@7.1.2
npm info linkStuff rimraf@2.6.2
npm verb linkBins rimraf@2.6.2
npm verb linkBins [ { rimraf: './bin.js' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans rimraf@2.6.2
npm info linkStuff fstream@0.1.31
npm verb linkBins fstream@0.1.31
npm verb linkMans fstream@0.1.31
npm info linkStuff unzip@0.1.11
npm verb linkBins unzip@0.1.11
npm verb linkMans unzip@0.1.11
npm info linkStuff pump@1.0.2
npm verb linkBins pump@1.0.2
npm verb linkMans pump@1.0.2
npm info linkStuff pumpify@1.3.5
npm verb linkBins pumpify@1.3.5
npm verb linkMans pumpify@1.3.5
npm info linkStuff xtend@4.0.1
npm verb linkBins xtend@4.0.1
npm verb linkMans xtend@4.0.1
npm info linkStuff tar-stream@1.5.4
npm verb linkBins tar-stream@1.5.4
npm verb linkMans tar-stream@1.5.4
npm info linkStuff tar-fs@1.16.0
npm verb linkBins tar-fs@1.16.0
npm verb linkMans tar-fs@1.16.0
npm info linkStuff through2@2.0.3
npm verb linkBins through2@2.0.3
npm verb linkMans through2@2.0.3
npm info linkStuff peek-stream@1.1.2
npm verb linkBins peek-stream@1.1.2
npm verb linkMans peek-stream@1.1.2
npm info linkStuff gunzip-maybe@1.4.1
npm verb linkBins gunzip-maybe@1.4.1
npm verb linkBins [ { 'gunzip-maybe': './bin.js' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/node_modules/.bin',
npm verb linkBins   false ]
npm verb linkMans gunzip-maybe@1.4.1
npm info linkStuff go-ipfs-dep@0.4.11
npm verb linkBins go-ipfs-dep@0.4.11
npm verb linkMans go-ipfs-dep@0.4.11
npm info linkStuff go-ipfs@0.4.11
npm verb linkBins go-ipfs@0.4.11
npm verb linkBins [ { ipfs: 'bin/ipfs' },
npm verb linkBins   '/Users/danny/.nvm/versions/node/v8.8.1/bin',
npm verb linkBins   true ]
npm verb linkMans go-ipfs@0.4.11
npm verb unlock done using /Users/danny/.npm/_locks/staging-c25489efd2f142c4.lock for /Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/.staging
npm verb stack Error: ENOENT: no such file or directory, chmod '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/bin/ipfs'
npm verb cwd /Users/danny
npm verb Darwin 16.7.0
npm verb argv "/Users/danny/.nvm/versions/node/v8.8.1/bin/node" "/Users/danny/.nvm/versions/node/v8.8.1/bin/npm" "install" "-g" "go-ipfs" "--verbose"
npm verb node v8.8.1
npm verb npm  v5.5.1
npm ERR! path /Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/bin/ipfs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/danny/.nvm/versions/node/v8.8.1/lib/node_modules/go-ipfs/bin/ipfs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
npm verb exit [ -2, true ]
```

Having the same here.

I'm trying to install as project dep and not global.

```
npm verb stack Error: ENOENT: no such file or directory, chmod '/Users/paulogr/dev/projects/dstatuspagejs/node_modules/go-ipfs/bin/ipfs'
npm verb cwd /Users/paulogr/dev/projects/dstatuspagejs
npm verb Darwin 16.7.0
npm verb argv "/Users/paulogr/.nvm/versions/node/v9.2.0/bin/node" "/Users/paulogr/.nvm/versions/node/v9.2.0/bin/npm" "install" "go-ipfs" "--verbose"
npm verb node v9.2.0
npm verb npm  v5.6.0
npm ERR! path /Users/paulogr/dev/projects/dstatuspagejs/node_modules/go-ipfs/bin/ipfs
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/paulogr/dev/projects/dstatuspagejs/node_modules/go-ipfs/bin/ipfs'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
npm verb exit [ -2, true ]
```

--- a/bin/ipfs
+++ b/bin/ipfs
@@ -1,1 +1,3 @@
-/Users/tmp/code/npm-go-ipfs/node_modules/go-ipfs-dep/go-ipfs/ipfs
+#!/usr/bin/env node
+
+console.log('IPFS is not installed.')

--- a/link-ipfs.js
+++ b/link-ipfs.js
@@ -1,22 +1,39 @@
 'use strict'
 
-var fs = require('fs')
-var path = require('path')
-var cproc = require('child_process')
-var version = require('./package.json').version
-version = version.replace(/-hacky[0-9]+/, '') // remove superfluous -suffix.
+const fs = require('fs')
+const path = require('path')
+const cproc = require('child_process')
+const isWin = process.platform === 'win32'
+const version = require('./package.json').version
+  .replace(/-hacky[0-9]+/, '') // remove superfluous -suffix.
 
 var depPath = require.resolve('go-ipfs-dep')
 depPath = path.dirname(path.dirname(depPath)) // walk up ../../
-const depBin = path.join(depPath, 'go-ipfs', 'ipfs')
-const localBin = path.join(__dirname, 'bin', 'ipfs')
+var depBin = path.join(depPath, 'go-ipfs', 'ipfs')
+var localBin = path.join(__dirname, 'bin', 'ipfs')
+
+if (isWin) {
+  depBin += '.exe'
+  localBin += '.exe'
+}
 
 if (!fs.existsSync(depBin)) {
   die('ipfs binary not found. maybe go-ipfs-dep did not install correctly?')
 }
 
-fs.unlinkSync(localBin)
+if (fs.existsSync(localBin)) {
+  fs.unlinkSync(localBin)
+}
+
 fs.symlinkSync(depBin, localBin)
+
+if (isWin) {
+  // On Windows, update the shortcut file to use the .exe
+  let cmdFile = path.join(__dirname, '..', '..', 'ipfs.cmd')
+
+  fs.writeFileSync(cmdFile, `@ECHO OFF
+"%~dp0\\node_modules\\go-ipfs\\bin\\ipfs.exe" %*`)
+}
 
 // test ipfs installed correctly.
 var result = cproc.spawnSync(localBin, ['version'])

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "install": "node link-ipfs.js",
-    "test": "tape test/*.js | tap-spec"
+    "test": "tape test/*.js | tap-spec",
+    "lint": "standard"
   },
   "bin": {
     "ipfs": "bin/ipfs"
@@ -33,6 +34,7 @@
   },
   "homepage": "https://github.com/ipfs/npm-go-ipfs",
   "devDependencies": {
+    "standard": "^11.0.1",
     "tap-spec": "^4.1.1",
     "tape": "^4.9.0"
   }

